extension/packages/hardhat/contracts/YourCollectible.sol

```diff
@@ -0,0 +1,171 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0 <0.9.0;
+
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "base64-sol/base64.sol";
+
+import "./HexStrings.sol";
+import "./ToColor.sol";
+
+//learn more: https://docs.openzeppelin.com/contracts/5.x/erc721
+
+// GET LISTED ON OPENSEA: https://testnets.opensea.io/get-listed/step-two
```

Comment:
Link is broken, I think it's better to just remove it

Reply:
Removed! Thanks!